### ISSUE TITLE
DEC Special Character Set (Line drawing)

### ISSUE THREAD

**Issue by kugland** [alacritty/alacritty#6095]
On commit 1dacc9918 the current table of the DEC Special Character Set has been introduced.

The current values deviate from the ones in XTerm. Gnome-terminal and derivatives, and also KDE’s Konsole follow XTerm in this.

| Character | Alactritty | XTerm |
|-------|-------|-------|
| `b`     | `'\t'` (Tab character) | `U+2409` (Symbol for horizontal tabulation) |
| `c`     | `'\u{000c}'` (Form feed) | `U+240C` (Symbol for form feed) |
| `d`    | `'\r'` (Carriage return) | `U+240D` (Symbol for carriage return) |
| `e`    | `'\n'` (Line feed) | `U+240A` (Symbol for line feed) |
| `i`     | `'\u{000b}'` (Vertical tabulation) | `U+240B` (Symbol for vertical tabulation) |

On Alacritty:
![alacritty](https://user-images.githubusercontent.com/1173932/171280810-1c2f3a25-e1b6-4cd5-900d-1477d8b6b298.png)

On XTerm:
![xterm](https://user-images.githubusercontent.com/1173932/171280660-3ec23574-a6df-414c-a65e-e390b4c6c6fc.png)

**Comment by chrisduerr** [comment]
Good catch, thanks. This bug is actually far older than you suggested: 62294771905917087c484bfc57e4f6ca547d89ba

I suppose that goes to show how rarely these parts of the character set are used.
